framework/src/wifi_manager/wifi_manager.c

```diff
@@ -35,7 +35,7 @@
 #undef nvdbg
 #define nvdbg printf
 
-#define NUM_OF_STATE_HANDLER 10 // the number of states
+#define NUM_OF_STATE_HANDLER 11 // the number of states
```

Comment:
Why don't you put this definition at the end of enum?
enum _wifimgr_state {
  WIFIMGR_UNINITIALIZD =0,
 ...
  NUM_OF_STATE_HANDER
}

Reply:
If applying above, line 361 will be replace as below.
const wifimgr_handler g_handler[] = {

Reply:
@juitem Change completed.